README.md

```diff
@@ -4,6 +4,63 @@ This repository contains DFT and spindynamics data computed at Uppsala universit
 
 ## Folder structure
 
+All simulation data is stored in the `data/` directory. The content of each
+dataset is structured as follows:
+
+```yaml
+chemical-formula_optional-description:
+  rspt:
+    common_rstp_input:
+      - atomdens
+      - kmap
+      - spts
+      - symcof
+      - symt.inp
+    gs_x:
+      - data
+      - hist OR out_MF
+      - out_last
+    gs_y:
+      - ...
+    gs_z:
+      - ...
+    Jij:
+      - green.inp-*
+      - out-*
+      - data
+  uppasd:
+    MC:
+      - jfile
+      - momfile
+      - posfile
+      - inpsd.dat
+      - M(T)  # output
+  README.md
```

Comment:
I would allow README.md files in the top level and also subdirectories. These can contain human-readable information, e.g. comments about some simulation details, ... The file will not be processed by the mammos-parser and should not contain properties that are present in intrinsic_properties.yaml.